object-search/app.py

```diff
@@ -72,51 +70,35 @@ def dryrun():
 @click.command()
 @click.option(
     '--task',
-    '-task',
     '-t',
-    type=click.Choice(['index', 'query', 'index_restful', 'query_restful', 'dryrun'], case_sensitive=False),
-)
+    type=click.Choice(['index', 'index-restful', 'query-restful', 'dry'], case_sensitive=False))
```

Comment:
If you change this (renaming and especially removing query without rest support), please update the readme of this example as well as it is mentioning those options.

Reply:
I returned to the underscore to be consistent with the other examples and updated the README
https://github.com/jina-ai/examples/pull/477/commits/16acf207013f5e0b288cb58c9367ad8dd5cb7301